src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.hxx

```diff
@@ -0,0 +1,276 @@
+                           const int edge2_index) const;
+
+  /** Check if a occ vertex lies on an occ edge.
+   * \param [in]  vertex_index   The index of the occ vertex.
+   * \param [in]  edge_index     The index of the occ edge.
+   * \return                    1 if vertex lies on edge, otherwise 0.
+   */
+  int
+  t8_geom_is_vertex_on_edge (const int vertex_index, 
+                             const int edge_index) const;
+
+  /** Check if a occ vertex lies on an occ edge.
+   * \param [in]  edge_index     The index of the occ vertex.
+   * \param [in]  face_index     The index of the occ edge.
+   * \return                    1 if vertex lies on edge, otherwise 0.
+   */
+  int
+  t8_geom_is_edge_on_face (const int edge_index, 
+                           const int face_index) const;
+
+  /** Check if a occ vertex lies on an occ face.
+   * \param [in]  vertex_index   The index of the occ vertex.
+   * \param [in]  face_index     The index of the occ face.
+   * \return                    1 if vertex lies on face, otherwise 0.
+   */
+  int
+  t8_geom_is_vertex_on_face (const int vertex_index, 
+                             const int face_index) const;
+
+  /** Retrieves the parameter of an occ vertex on an occ edge.
+   *  The vertex has to lie on the edge.
+   * \param [in]  vertex_index   The index of the occ vertex.
+   * \param [in]  edge_index     The index of the occ edge.
+   * \param [out] edge_param     The parameter of the vertex on the edge.
+   */
+  void
+  t8_geom_get_parameter_of_vertex_on_edge(const int vertex_index, 
+                                          const int edge_index, 
+                                          double* edge_param) const;
+
+  /** Retrieves the parameters of an occ vertex on a occ face.
+   *  The vertex has to lie on the face.
+   * \param [in]  vertex_index   The index of the occ vertex.
+   * \param [in]  face_index     The index of the occ face.
+   * \param [out] face_params    The parameters of the vertex on the face.
+   */                                   
+  void 
+  t8_geom_get_parameters_of_vertex_on_face(const int vertex_index, 
+                                           const int face_index, 
+                                           double* face_params) const;
+
+  /** Converts the parameter of an occ edge to the corresponding parameters on an occ face.
+   *  The edge has to lie on the face.
+   * \param [in]  edge_index     The index of the occ edge.
+   * \param [in]  face_index     The index of the occ face.
+   * \param [in]  edge_param     The parameter on the edge.
+   * \param [out] face_params    The corresponding parameters on the face.
+   */   
+  void
+  t8_geom_edge_parameter_to_face_parameters(const int edge_index, 
+                                            const int face_index, 
+                                            const double edge_param, 
+                                            double* face_params) const;
+
+private:
```

Comment:
Doxygen inline comments (comments that concern the current line and are written at the end) must have `/**<` at the beginning.
So please change them.
see https://doxygen.nl/manual/docblocks.html (Putting documentation after members)

Reply:
resolved in 4cbc1b11